extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php

```diff
@@ -1,14 +1,28 @@
 <nav class="global-navigation" id="globalNavigation">
 	<div class="page-width">
 		<div class="global-navigation-item wikia-logo-container" id="hubsEntryPoint">
-			<a href="<?= htmlspecialchars( $centralUrl ) ?>" class="global-navigation-link wikia-logo" rel="nofollow"><img src="<?= $wg->BlankImgUrl ?>" height="24" width="91" alt="<?= wfMessage('venus-wikia')->escaped() ?>" title="<?= wfMessage('venus-wikia')->escaped() ?>"></a>
+			<a href="<?= htmlspecialchars( $centralUrl ) ?>"
+				class="global-navigation-link wikia-logo"
+				rel="nofollow"
+				data-id="wikia-logo">
+				<img src="<?= $wg->BlankImgUrl ?>"
+				     height="24"
+				     width="91"
+				     alt="<?= wfMessage('venus-wikia')->escaped() ?>"
+				     title="<?= wfMessage('venus-wikia')->escaped() ?>">
+			</a>
 			<?= $app->renderView('GlobalNavigation', 'hubsMenu') ?>
```

Comment:
Since you're cleaning up this file would you mind adding spacing to this line, 24, and 30? That'll fix all the spacing violations in here.